geniza/corpus/management/commands/import_iiif_urls.py

```diff
@@ -0,0 +1,85 @@
+import csv
+import re
+import logging
+from collections import defaultdict
+
+from django.core.management.base import BaseCommand
+
+from geniza.corpus.models import Fragment
+
+
+# logging config: use levels as integers for verbosity option
+logger = logging.getLogger("import")
+logging.basicConfig()
+LOG_LEVELS = {0: logging.ERROR, 1: logging.WARNING, 2: logging.INFO, 3: logging.DEBUG}
+
+
+class Command(BaseCommand):
+    """Given a CSV of fragments and IIIF URLs, add those IIIF urls to their respective fragments in the database"""
+
+    def __init__(self, *args, **options):
+        self.csv_path = options.get("csv")
+        self.overwrite = options.get("overwrite")
+        self.dryrun = options.get("dryrun")
+        self.stats = defaultdict(int)
+
+    def add_arguments(self, parser):
+        parser.add_argument("csv", type=str)
+        parser.add_argument("-o", "--overwrite", action="store_true")
+        parser.add_argument("-d", "--dryrun", action="store_true")
+
+    def handle(self, *args, **options):
+        self.csv_path = options.get("csv")
+        self.overwrite = options.get("overwrite")
+        self.dryrun = options.get("dryrun")
+
+        with open(self.csv_path) as f:
+            csvreader = csv.DictReader(f)
+            for row in csvreader:
```

Comment:
should probably bail out with an error if the csv headers aren't as expected (I say this since I just tried it with the old csv with wrong headers)